build/nodes-and-endpoints/join-mainnet.md

```diff
@@ -261,139 +261,7 @@ systemctl stop core-1.service
 systemctl restart core-1.service
 ```
 
-
 ## Upgrade the Node's Software
```

Comment:
Shall we also split out the 'run the node in the background' paragraph?

Reply:
We can do this and link the same common page in both Mainnet & Testnet pages. But at the same time, this will increase number of small pages in "nodes and endpoints" section. Please let me know your opinion.